### ISSUE TITLE
Undefined symbol "c_plshade1"

### ISSUE THREAD

**Issue by eserte** [PDLPorters/pdl-graphics-plplot#6]
On some of my smoker systems I see the following failure:
```
Can't load '/usr/home/cpansand/.cpan/build/2019122820/PDL-Graphics-PLplot-0.73-2/blib/arch/auto/PDL/Graphics/PLplot/PLplot.so' for module PDL::Graphics::PLplot: /usr/home/cpansand/.cpan/build/2019122820/PDL-Graphics-PLplot-0.73-2/blib/arch/auto/PDL/Graphics/PLplot/PLplot.so: Undefined symbol "c_plshade1" at /usr/perl5.28.0p/lib/5.28.0/amd64-freebsd/DynaLoader.pm line 193.
Compilation failed in require at t/plplot.t line 11.
BEGIN failed--compilation aborted at t/plplot.t line 11.
t/plplot.t ................ 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't load '/usr/home/cpansand/.cpan/build/2019122820/PDL-Graphics-PLplot-0.73-2/blib/arch/auto/PDL/Graphics/PLplot/PLplot.so' for module PDL::Graphics::PLplot: /usr/home/cpansand/.cpan/build/2019122820/PDL-Graphics-PLplot-0.73-2/blib/arch/auto/PDL/Graphics/PLplot/PLplot.so: Undefined symbol "c_plshade1" at /usr/perl5.28.0p/lib/5.28.0/amd64-freebsd/DynaLoader.pm line 193.
Compilation failed in require at t/plplot_library_tests.t line 6.
BEGIN failed--compilation aborted at t/plplot_library_tests.t line 6.
t/plplot_library_tests.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by cheese** [comment]
Met the same issue on Fedora when building this package as RPM.

**Comment by cheese** [comment]
After search the plplot sources, I found plshade1 deprecated as of plplot-5.14.0.
http://plplot.sourceforge.net/doxygen/html/plshade_8c_source.html

**Comment by mohawk2** [comment]
This has now been fixed (including needing to fix returning C pointers as `indx` not `int`) and released as 0.75.

Thank you @eserte as always! And @cheese please could you try this and report any problems?

Closing this, but please reopen if you find any further problems with this issue, or open a new one with any others.
